.github/workflows/test-all-packages.yml

```diff
@@ -296,18 +304,32 @@ jobs:
         path: .
         key: ${{ runner.os }}-${{ matrix.node-version }}-built-${{ github.sha }}
     - uses: actions/checkout@v1
+      with:
+        submodules: 'true'
       if: steps.built.outputs.cache-hit != 'true'
     - name: yarn install
       run: yarn install
       if: steps.built.outputs.cache-hit != 'true'
+    - name: install XS dependencies
+      working-directory: ./packages/xs-vat-worker
+      run: yarn install:xs-lin
     - name: yarn build
       run: yarn build
       if: steps.built.outputs.cache-hit != 'true'
+    - name: yarn build-xs-worker
+      working-directory: ./packages/xs-vat-worker
+      run: yarn build:xs-lin
```

Comment:
And this one too.  (You didn't change the boilerplate that all the tests use except for the `with: submodules: 'true'`.)